commands/auth.go

```diff
@@ -127,10 +136,32 @@ func loginCmdF(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	passwordFile, _ := cmd.Flags().GetString("password-file")
+	if password != "" && passwordFile != "" {
+		return errors.New("can not use two passwords at the same time")
+	}
+	if passwordFile != "" {
+		b, err2 := ioutil.ReadFile(passwordFile)
+		if err2 != nil {
+			return err2
+		}
+		password = strings.TrimSpace(string(b))
```

Comment:
Wondering if this should become something like a `readStringFromFile` util function.

Reply:
Makes sense 👍